app/src/androidTest/java/net/osmtracker/Layouts/TestUtils.java

```diff
@@ -0,0 +1,116 @@
+package net.osmtracker.Layouts;
```

Comment:
what about moving this file to a package `net.osmtracker.util` ?

Reply:
Yes, I'll change it